### PR TITLE
RHOAIENG-5466: Add KServeV1HTTPPredictionProvider

### PR DESCRIPTION
**Pull request by ruivieira**
[RHOAIENG-5466](https://issues.redhat.com/browse/RHOAIENG-5466):

This PR allows to call KServe models using HTTP and JSON payloads and the v1 protocol.

### CODE REVIEW

File: explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/KServeDatatype.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
-package org.kie.trustyai.connectors.kserve.v2;
+package org.kie.trustyai.connectors.kserve;
```
Comment (danielezonca) [inline explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/KServeDatatype.java:1]: Are you sure `v2` data types are not going to diverge from v1 data types?

**Review by danielezonca — APPROVED** [review]
Approving with minor comment

File: explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/KServeDatatype.java (hunk 1 above)
Comment (ruivieira) [reply explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/KServeDatatype.java:1]: My understanding is that `v1` and `v2` are standardised so they won't change. We might have `v3` in the future of course, we'll just have to re-evaluate when/if that happens.

**Pull request merged**